protected/humhub/modules/admin/controllers/SettingController.php

```diff
@@ -315,6 +321,95 @@ public function actionLogs()
         ]);
     }
 
+    public function actionTopics()
+    {
+        $model = new Topic();
+        $suggestGlobalConversion = false;
+
+        if ($model->load(Yii::$app->request->post())) {
+            $model->on($model::EVENT_GLOBAL_CONVERSION_SUGGESTION, function () use (&$suggestGlobalConversion) {
```

Comment:
Why do we need the event? Can't we just use the DB query to check whether the topic is already in use?

Reply:
it's for performance purposes, so we call ContentTag::validateUnique once, with database solution we'll need to call this method/query twice

Reply:
Ok. Need a closer look.